Games/types/Mafia/roles/Mafia/Mummy.js

```diff
@@ -0,0 +1,17 @@
+const Role = require("../../Role");
+
+module.exports = class Mummy extends Role {
+
+    constructor(player, data) {
+        super("Mummy", player, data);
+        this.alignment = "Mafia";
+        this.cards = ["VillageCore", "WinWithMafia", "MeetingMafia", "NightRoleBlocker", "KillVisitorsWhileDead"];

```

Comment:
this is a mechanic question -- is there really a use for "kill visitors while dead" in mummy

It makes mummy a very beefed role, able to join mafia meeting, roleblock when dead and kill visitors when dead

don't really forsee a situation when it would get visited when dead, especially if it's role is revealed when dead

Reply:
That was an aspect from mummy that was taken from EM, but I believe that was the key drawing factor compared to the roleblocking aspect. Main usage would be as a counter to mortician I believe.

Reply:
Will turn mummy into mummy and poltergeist